test/mock.ts

```diff
@@ -6,16 +6,16 @@ import { Result, Job } from "../src/job";
 export function mockProject(): Project {
   return {
     id: "brigade-c0ff33544b459e6ac0ffee",
```

Comment:
This ID will might need to change.  It is a hash of the project name.

Reply:
Thanks; updated with what `brig project create` gave me (off of branch in https://github.com/brigadecore/brigade/pull/850)